includes/class-wc-stripe-api.php

```diff
@@ -16,6 +16,27 @@ class WC_Stripe_API {
 	const ENDPOINT           = 'https://api.stripe.com/v1/';
 	const STRIPE_API_VERSION = '2024-06-20';
 
+	/**
+	 * The Invalid API Keys cache key.
+	 *
+	 * @var string
+	 */
+	const INVALID_API_KEYS_CACHE_KEY = 'invalid_api_keys_rate_limit';
```

Comment:
Here and below for the other new `const`s: I think we should specify visibility explicitly. It looks like the cache key and threshold should be `public`, while delay can be `private` or `protected`.

Reply:
Good idea... fixed in [c144129](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4497/commits/c144129360e40c4be5ce9c26dab917ef511396f4)